cordova/cordova-webapp-android-tests/webapp/comm.py

```diff
@@ -47,7 +48,7 @@
 
 
 def setUp():
-    global ARCH, MODE, VERSION, device
+    global ARCH, MODE, VERSION, device, CROSSWALK_VERSION
```

Comment:
Pls rename "VERSION" as "CORDOVA_VERSION" to be consistent with "CROSSWALK_VERSION"